build.sbt

```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   "edu.arizona.sista" % "processors" % "3.0" classifier "models",
   "org.slf4j" % "slf4j-api" % "1.7.2",
   "org.slf4j" % "slf4j-log4j12" % "1.7.2",
-  "org.scalatest" %% "scalatest" % "1.9.1" % "test",
+  "org.scalatest" %% "scalatest" % "2.2.6" % "test",
```

Comment:
Alright - fighting with class path hell doesn't seem like a productive use of time right now. I'm going to revert and worst case we can factor out the convolution benchmarks into a separate class.